app/components/embed/legacy_pdf_component.html.erb

```diff
@@ -2,28 +2,48 @@
   <% # removing 5 pixels from the body height to account for being embedded %>
   <div class='sul-embed-body sul-embed-pdf'>
-    <button aria-label="Zoom in" class="button zoom-in">+</button>
-    <button aria-label="Zoom out" class="button zoom-out">-</button>
-    <button aria-label="Previous page" class="button prev-page">
-      <i class="sul-i-arrow-left-8"></i>
-    </button>
-    <button aria-label="Next page" class="button next-page">
-      <i class="sul-i-arrow-right-8"></i>
-    </button>
-    <div id="pdf-viewer" class="pdf-viewer"
-      data-pdf-url="<%= viewer.pdf_files.first %>"
-      data-location-restricted="<%= viewer.all_documents_location_restricted? %>"
-    >
-      <div class="loading-spinner"></div>
-      <canvas></canvas>
-      <button
-        aria-label="Exit full screen"
-        class="btn sul-i-close sul-close-fullscreen-button"
-        id="close-full-screen-button">
-      </button>
-    </div>
-    <% # NOTE: Remove legacy_pdf_viewer.js file when this component is removed %>
-    <%= javascript_pack_tag('legacy_pdf_viewer') %>
+    <% if viewer.available? %>
+        <button aria-label="Zoom in" class="button zoom-in">+</button>
+        <button aria-label="Zoom out" class="button zoom-out">-</button>
+        <button aria-label="Previous page" class="button prev-page">
+          <i class="sul-i-arrow-left-8"></i>
+        </button>
+        <button aria-label="Next page" class="button next-page">
+          <i class="sul-i-arrow-right-8"></i>
+        </button>
+        <div id="pdf-viewer" class="pdf-viewer"
+          data-pdf-url="<%= viewer.pdf_files.first %>"
+          data-location-restricted="<%= viewer.all_documents_location_restricted? %>"
+        >
+          <div class="loading-spinner"></div>
+          <canvas></canvas>
+          <button
+            aria-label="Exit full screen"
+            class="btn sul-i-close sul-close-fullscreen-button"
+            id="close-full-screen-button">
+          </button>
+          <button aria-label="Next page" class="button next-page">
+            <i class="sul-i-arrow-right-8"></i>
+          </button>
+          <div id="pdf-viewer" class="pdf-viewer"
+            data-pdf-url="<%= viewer.pdf_files.first %>"
+            data-location-restricted="<%= viewer.all_documents_location_restricted? %>"
+          >
+            <div class="loading-spinner"></div>
+            <canvas></canvas>
+            <button
+              aria-label="Exit full screen"
+              class="sul-embed-btn sul-i-close sul-close-fullscreen-button"
+              id="close-full-screen-button">
+            </button>
+          </div>
+          <% # NOTE: Remove legacy_pdf_viewer.js file when this component is removed %>
+          <%= javascript_pack_tag('legacy_pdf_viewer') %>
+      <% else %>
+          <div style="padding: 10%;">
+            <strong><%= I18n.t('restrictions.not_accessibile') %></strong>
+          </div>
+      <% end %>
   </div>
```

Comment:
these are for the legacy PDF viewer component, same changes below are for the new browser based PDF viewer component